src/plot_api/plot_config.js

```diff
@@ -114,6 +114,12 @@ var configAttributes = {
         }
     },
 
+    editSelection: {
+        valType: 'boolean',
+        dflt: true,
+        description: 'Disables moving selections.'
```

Comment:
Either "Enables moving selections" or "Setting this to false disables moving selections."

Reply:
[a8c7fa8](https://github.com/plotly/plotly.js/pull/6285/commits/a8c7fa86dcff7c597260216fc306b99134a6ef4f)